core/src/main/resources/IAF_WebControl/IAF_DatabaseChangelog.xml

```diff
@@ -128,5 +128,12 @@ TagDatabase
 		<!-- in Quartz the JobKey is defined by name and group, hence this being the primairy key constraint in the database as well -->
 		<addPrimaryKey tableName="IBISSCHEDULES" columnNames="JOBNAME, JOBGROUP" constraintName="PK_IBISSCHEDULES"/>
 	</changeSet>
+	
+	<changeSet id="IAF_Util:7" author="Ali Sihab Akcan">
+		<comment>Add description column to IBISSCHEDULES Table</comment>
+		<addColumn tableName="IBISSCHEDULES" >
+			<column afterColumn="MESSAGE" name="DESCRIPTION"  type="java.sql.Types.VARCHAR(255)"/> 
+		</addColumn>
+	</changeSet>
```

Comment:
I think it's wise to add a default (for example `null`) to the description field. If you want to add a column without default value to an already populated table you will get sql errors

Reply:
I think having defaultValue is required when we have a nullable=false constraint on the column for the existing rows to have a real value. With the above change, the column(nullable) will be added and the existing rows will have NULL value by default